lib/plugin/videoroom/errors.ex

```diff
@@ -0,0 +1,34 @@
+defmodule Janus.Plugin.VideoRoom.Errors do
+  # TODO: resolve handling errors properly to handle all possible cases
+
+  @errors %{
+    # code to atom
+    426 => :no_such_room,
+    427 => :room_already_exists,
+    428 => :no_such_feed,
+    # atom to code
+    :no_such_room => 426,
+    :room_already_exists => 427,
+    :no_such_feed => 428
+  }
+
+  @spec error(integer()) :: {:error, atom()}
+  def error(code) do
+    {:error, Map.fetch!(@errors, code)}
```

Comment:
I get that you don't want to handle all errors now, but it would be nice if this fetch not crashed in case of getting unhandled error. I got `INVALID_ELEMENT` 430, and plugin crashed. use a regular get with default value `:unknown_error` or `:unsupported_error`